### PR TITLE
[FLINK-20907][docs] replaces deprecated syntax in Table API doc

### PR DESCRIPTION
**Pull request by sv3ndk**
## What is the purpose of the change

*replacement of deprecated syntax in Table API documentation*

## Brief change log

* removes from documentation statements using this syntax: `tableEnv.connect(...).createTemporaryTable("table1");`, which is [marked as depracated](https://github.com/apache/flink/blob/97bfd049951f8d52a2e0aed14265074c4255ead0/flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/bridge/java/StreamTableEnvironment.java#L629)
* renames `Table` instance `tapiResult` into `table2`, since the "Result" suffix was introducing confusion with possible instances of `TableResult` , which are returned by other methods of `tableEnv` 

## Verifying this change

ran `./build_docs.sh -p` and checked the result visually

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

File: docs/dev/table/common.md [hunk 1]
```diff
@@ -51,18 +51,18 @@ All Table API and SQL programs for batch and streaming follow the same pattern.
 // create a TableEnvironment for specific planner batch or streaming
 TableEnvironment tableEnv = ...; // see "Create a TableEnvironment" section
 
-// create a Table
-tableEnv.connect(...).createTemporaryTable("table1");
+// create an input Table
+tableEnv.executeSql("CREATE TEMPORARY TABLE table1 ... with ( 'connector' = ... )");
 // register an output Table
-tableEnv.connect(...).createTemporaryTable("outputTable");
+tableEnv.executeSql("CREATE TEMPORARY TABLE outputTable ... with ( 'connector' = ... )");
```
Comment (wuchong) [inline docs/dev/table/common.md:57]: nit: could you use the upper case `WITH` instead of `with`?

**Review by wuchong — COMMENTED** [review]
Thanks @sv3ndk , the change looks good to me. I only left a minor comment.

File: docs/dev/table/common.md (hunk 1 above)
Comment (sv3ndk) [reply docs/dev/table/common.md]: Yes, good point, thanks for the feedback. 
Done.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**